### PR TITLE
Fix width height issues in widgets

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

**Comment by jmthomas** [comment]
I hate how we came up with these screen definitions and scattered the parameters around. Any time we have a multi-widget (like LABELVALUERANGEBAR is the worst) it seems like we just picked and chose parameters to pass on to to the sub-widgets. Not sure the best path forward now because it would change screen definitions but this is pretty hard to maintain the way it is.

**Comment by jmthomas** [comment]
closes #157

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST/screens/ground.txt [hunk 1]
```diff
@@ -6,12 +6,13 @@ VERTICAL
...
   SETTING BACKCOLOR white
 
-  CANVAS 650 700
+  CANVAS 650 600
     # The code inside <% %> is just ruby
-    <% 11.times do |x| %>
-      # Create a dot at various y locations. The <%= %> adds the expression to the file
-      CANVASDOT 5 <%= 50 * x + 50 %> BLUE 5
-    <% end %> # Close our block from above
+    # The code inside  is just ruby
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST/screens/ground.txt:11]: This comment doesn't make sense anymore

**Review by ryanmelt — CHANGES_REQUESTED** [review]
Looks good. Just a minor comment and playwright needs to pass.

**Pull request merged**